src/query/functions/aggregation/function_test.go

```diff
@@ -34,15 +34,15 @@ import (
 )
 
 func TestCountWithAllValues(t *testing.T) {
-	values, bounds := test.GenerateValuesAndBounds(nil, nil)
-	block := test.NewBlockFromValues(bounds, values)
+	vals, b := test.GenerateValuesAndBounds(nil, nil)
```

Comment:
Pass in values instead of `nil`?

Reply:
Deleted these two tests since they're covered in the tests below